Makefile

```diff
@@ -242,6 +242,7 @@ docker:
 	${CONTAINER_CPUSET} \
 	--pid=host \
 	--privileged \
+	--cgroupns=host \
```

Comment:
what does this do?

Reply:
by default the docker namespace is restricted from managing cgroups internally. this flag grants the container the necessary permissions for host cgroup management.

https://www.whexy.com/posts/cgroup-inside-containers